Automattic/wp-calypso
# Plugins: Use constants for status actions

In prior plugins reduxification work, as well as in the old flux store, we often use a string representation of status actions instead of the already defined constants. This PR updates the code so it always uses the constants.

Part of #24180.

#### Changes proposed in this Pull Request

* Plugins: Use constants for status actions

#### Testing instructions

* Smoke test plugins activation, deactivation, installation, removal, autoupdate toggling (in bulk and single mode) and verify global and inline notices still work the same way.
* Verify all tests pass.

#### Note

ESLint warnings are expected in one of the files - we're intentionally not fixing them here.
